### PR TITLE
Document Dart team publishers

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

**Review by kevmoo — APPROVED** [review]

**Review by kwalrath — CHANGES_REQUESTED** [review]
I've rewritten this (so much easier to rewrite than to write from scratch!).

**Comment by kwalrath** [comment]
I've rewritten this to put everything into one table and to swap the orientation of the table. Also, it appears that dart.dev/packages redirects to pub.dev! (I probably would've renamed the page anyway.)

My version:
https://kw-www-dartlang-1.firebaseapp.com/dart-team-packages

Mit's original (please make sure I didn't miss anything):
https://kw-www-dartlang-1.firebaseapp.com/packages-ORIG

**Comment by mit-mit** [comment]
That looks great Kathy! But let's wait for a review by Lasse and Nate before we merge.

**Review by kwalrath — APPROVED** [review]

File: src/dart-team-packages.md
Comment (natebosch) [inline src/dart-team-packages.md:3]: [nit] 
```suggestion
description: The Dart team publishes packages under the verified publishers dart.dev, tools.dart.dev, and labs.dart.dev.
```

**Review by natebosch — APPROVED** [review]

**Review by lrhn — APPROVED** [review]
SGTM

**Pull request merged**
